### PR TITLE
Update network.rst:

### PR DESCRIPTION
**Pull request by Mrt134**
- Update text to reflect angulargui.
- Rework text slightly to clarify one aspect of lagg creation.
- Comment out lagg members documentation until redmine issue 31701 is resolved.
- link testing: fix any issues with external links.
- Igor review: resolve any issues.
- HTML build test: no issues.

### CODE REVIEW

File: userguide/network.rst [hunk 1]
```diff
@@ -164,16 +162,17 @@ or editing a manually configured interface.
...
 shows the screen that opens on clicking
 :menuselection:`Interfaces --> Add Interface`.
 :numref:`Table %s <net_interface_config_tab>`
 summarizes the configuration options shown when adding an interface or
-editing an already configured interface. Note that if any changes to
-this screen require a network restart, the screen will turn red when
-the :guilabel:`OK` button is clicked and a pop-up message will point
-out that network connectivity to the %brand% system will be
-interrupted while the changes are applied.
+editing an already configured interface.
```
Comment (wblock) [inline userguide/network.rst]: s/already configured/existing/

File: userguide/network.rst [hunk 2]
```diff
@@ -541,11 +523,12 @@ Creating a Link Aggregation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 **Before** creating a link aggregation, double-check that no
```
Comment (wblock) [inline userguide/network.rst]: Oddly worded, and not sure about the bold, but still: "**Before** creating a link aggregation, make sure that all interfaces to be used in the lagg have not been manually configured in :menuselection:`Network --> Interfaces`.

File: userguide/network.rst [hunk 3]
```diff
@@ -541,11 +523,12 @@ Creating a Link Aggregation
...
-interfaces have been manually configured in
-:menuselection:`Network --> Interfaces --> View Interfaces`.
+interfaces to be used with the lagg are manually configured in
+:menuselection:`Network --> Interfaces`.
 
-If any manually-configured interfaces exist, delete them as
-**lagg creation fails if any interfaces are manually configured**.
+If any manually-configured interfaces to be included with the lagg
```
Comment (wblock) [inline userguide/network.rst]: s/with/in/

File: userguide/network.rst [hunk 4]
```diff
@@ -541,11 +523,12 @@ Creating a Link Aggregation
...
+exist, delete them as **lagg creation fails if any of the included**
```
Comment (wblock) [inline userguide/network.rst]: Split sentence: "... delete them. **lagg creation..."

File: userguide/network.rst [hunk 5]
```diff
@@ -541,11 +523,12 @@ Creating a Link Aggregation
...
+**interfaces are manually configured**.
```
Comment (wblock) [inline userguide/network.rst]: s/are/have been/

File: userguide/network.rst [hunk 6]
```diff
@@ -698,58 +676,47 @@ The configurable options are summarized in
...
+ the smallest value is used for the MTU of the entire lagg.
 
-.. note:: A reboot is required after changing the MTU to create a
+ .. note:: A reboot is required after changing the MTU to create a
    jumbo frame lagg.
 
 
-To see if the link aggregation is properly load balancing, run this
-command from :ref:`Shell`:
+ To see if the link aggregation is properly load balancing, run this
```
Comment (wblock) [inline userguide/network.rst]: "Link aggregation load balancing can be tested with:" (followed by just the code block).

File: userguide/network.rst [hunk 7]
```diff
@@ -863,30 +831,31 @@ summarizes the configurable fields.
...
-The parent interface of a VLAN must be up, but it can have an IP
-address or it can be unconfigured, depending upon the requirements of
-the VLAN configuration. This makes it difficult for the GUI to do the
-right thing without trampling the configuration. To remedy this, after
-adding the VLAN, go to
+The parent interface of a VLAN must be up, but it can either have an IP
+address or be unconfigured, depending upon the requirements of the VLAN
+configuration. This makes it difficult for the GUI to do the right thing
+without trampling the configuration. To remedy this, after adding the
+VLAN, go to
```
Comment (wblock) [inline userguide/network.rst]: "To remedy this, add the VLAN, then select"

File: userguide/network.rst [hunk 8]
```diff
@@ -863,30 +831,31 @@ summarizes the configurable fields.
...
 :menuselection:`Network --> Interfaces --> Add Interface`.
 Select the parent interface from the :guilabel:`NIC` drop-down menu
```
Comment (wblock) [inline userguide/network.rst]: s/Select/Choose/

File: userguide/network.rst [hunk 9]
```diff
@@ -896,8 +865,7 @@ configured using the rest of the options in the
 
 #ifdef freenas
 .. warning:: Creating a vlan will cause network connectivity to be
-   interrupted.  Accordingly, the GUI will provide a warning and an
-   opportunity to cancel the vlan creation.
+   interrupted.
```
Comment (wblock) [inline userguide/network.rst]: Reword, but the UI now has network interruption warnings, too.

**Comment by dlavigne** [comment]
Remember to create a master PR for any text improvements.

**Pull request merged**